.github/workflows/publish.yml

```
@@ -0,0 +1,39 @@
+name: Publish to RubyGems
+
+on:
+  release:
+    types:
+      - [ created ]
+  push:
+    tags:
+      - "v*"
+
```

let's also add explicit permissions here:
```suggestion
permissions:
  contents: read
```

---

Done. 😄